docs/search/search_operators.rst

```diff
@@ -1,6 +1,262 @@
 .. vale off
 
-Search operators
+Searching Mautic
 ################
 
-.. vale on
\ No newline at end of file
+.. vale on
+
+Search operators and filters
+============================
+
+Mautic offers a variety of search operators and filters for drilling down into relevant resources. You can find the available search filters and operators by clicking on the button with a question mark next to the search input.
+The search filters for that entity aren't available if such a button is missing.
+
+.. image:: images/contacts-search.png
+   :align: center
+   :alt: Mautic Contact search
+   
+|
+
+Mautic also has a 'global search' feature. In the top left-hand corner, click the magnifying glass icon next to the Mautic logo/notifications icon. This will open a search input where you can search across multiple different entities.
+
+.. image:: images/global-search.png
+   :align: center
+   :alt: Mautic global search
+
+|
+
+Search operators
+----------------
+
+Here are some search operators you can use:
+
+* ``+`` plus sign - Search for the exact string, for example, if you search for +admin, then administrator won't match.
+
+* ``!`` exclamation mark - Not equals string
+  
+* ``" "`` double quotes - Search by phrase
+  
+* ``( )`` parentheses - Group expressions together.
+  
+* ``OR`` - By default the expressions joins as ``AND`` statements. Use the OR operator to change that.
+
+* ``%`` - Use the % as a wildcard to search for specific names or values in a phrase for example, to find all Companies with the word ‘Technologies’ then type %technologies%
+  
+Search filters
+--------------
```

Comment:
This should be H3 (===) rather than H4 (----)This should be H3 (===) rather than H4 (----)